<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskListGUI</class>
 <widget class="QWidget" name="TaskListGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskListGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>10000</width>
     <height>10000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);</string>
   </property>
   <widget class="QWidget" name="leftwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>171</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton:checked {
      border-image: url(:/XMSDiskClientGui/Resources/task_bak.png);
font: 11pt &quot;Adobe Arabic&quot;;
	color: rgb(6, 181, 253);
}

QPushButton:!checked {
font: 11pt &quot;Adobe Arabic&quot;;
	color: rgb(0, 0, 0);
}
QWidget
{
background-color: rgb(249, 250, 251);
}</string>
    </property>
    <widget class="QPushButton" name="downButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/XMSDiskClientGui/Resources/img/down1.png) off, url(:/XMSDiskClientGui/Resources/img/down2.png) on;</string>
     </property>
     <property name="text">
      <string>正在下载</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>42</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/XMSDiskClientGui/Resources/img/upload1.png) off, url(:/XMSDiskClientGui/Resources/img/upload2.png) on;</string>
     </property>
     <property name="text">
      <string>正在上传</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="okButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>84</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-icon: url(:/XMSDiskClientGui/Resources/img/succ1.png) off, url(:/XMSDiskClientGui/Resources/img/succ2.png) on;</string>
     </property>
     <property name="text">
      <string>传输完成</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="downlabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>13</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;思源黑体 CN Normal&quot;;
color: rgb(255, 0, 4);
background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>(0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="oklabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>91</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;思源黑体 CN Normal&quot;;
color: rgb(255, 0, 4);
background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>(0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="uplabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>51</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;思源黑体 CN Normal&quot;;
color: rgb(255, 0, 4);
background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string>(0)</string>
     </property>
    </widget>
    <zorder>downButton</zorder>
    <zorder>okButton</zorder>
    <zorder>downlabel</zorder>
    <zorder>oklabel</zorder>
    <zorder>upButton</zorder>
    <zorder>uplabel</zorder>
   </widget>
   <widget class="QTableWidget" name="tasktableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>831</width>
      <height>611</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section {
     padding-left: 0px;
 }
QTableWidget                   
{
    background:#FFFFFF;           
    border:none;                   
 
    font-size:14px;                
    font-family:&quot;Microsoft YaHei&quot;;
    color:#666666;   

 selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,
                                  stop: 0 #FF92BB, stop: 1 white);              
}
QTableWidget::item                
{
    background:#ffffff; 
    spacing: 3px;
}

QTableView::item::selected    
{         
  color:#000000;             
  background:#e6f6ff; 
}


QHeaderView::section {
background-color:#FFFFFF;
  border: 1px solid #f0f0f0 ;

}


QTableView::item {

      border-bottom-width: 1px;
      border-style: solid;
      border-color: #f0f0f0;
      padding-left: 10px;
	  border-right: none;
border-left: none;
border-top: none;
	

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>55</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>文件名</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="disk_client_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tasktableWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>TaskListGUI</receiver>
   <slot>Select(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>TaskListGUI</receiver>
   <slot>DownTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>990</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upButton</sender>
   <signal>clicked()</signal>
   <receiver>TaskListGUI</receiver>
   <slot>UpTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>998</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>TaskListGUI</receiver>
   <slot>OkTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>998</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>TaskListGUI</receiver>
   <slot>RefreshUploadTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>-26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Select(QModelIndex)</slot>
  <slot>DownTask()</slot>
  <slot>UpTask()</slot>
  <slot>OkTask()</slot>
  <slot>RefreshUploadTask()</slot>
  <slot>RefreshDownloadTask()</slot>
 </slots>
</ui>
